api/v1alpha1/condition_consts.go

```diff
@@ -37,3 +37,21 @@ const (
 	// DeletedReason (Severity=Info) documents a condition not in Status=True because the underlying object was deleted.
 	DeletedReason = "Deleted"
 )
+
+// Conditions related to the VirtualMachineImages
+const(
+	// VirtualMachineImageOsTypeSupportedCondition denotes that the OS type in the virtual machine image object is
+	// supported by VMService or not. e.g. Image with windows OS type is not supported by VM Service.
```

Comment:
Can we consistently use either "VMService" or "VM Service"?

I think the "or not" can be omitted

"windows" might need to be capitalized, since we do for "Linux" below